index.js

```diff
@@ -1,7 +1,20 @@
 const readline = require('readline-sync');
+const calc = require('./calculator');
 
 console.log('Welcome to the calculator!');
 
-console.log('Please enter some input:');
-const response = readline.prompt();
-console.log('You entered: ' + response);
+console.log('Enter the operator:');
+const operator = readline.prompt();
+
+console.log(`How many numbers do you want to ${operator} ?`);
+const c = parseInt(readline.prompt());
```

Comment:
Could we have a slightly more clear variable name here? Possibly something like "numberOfNumbersToOperateOn" or something slightly more concise and clear than that if you can think of one! 😅